src/test/java/com/trello/boards/NewBoardTest.java

```diff
@@ -0,0 +1,106 @@
+package com.trello.boards;
+
+import com.trello.BoardPage;
+import com.trello.LogInPage;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.interactions.Actions;
+import org.testng.Assert;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+public class NewBoardTest {
+
+    private ChromeDriver driver;
+
+    @BeforeClass
+    public void webDriver() {
+        System.setProperty("webdriver.chrome.driver", "C://Users//Office//Downloads//chromedriver_win32 (1)//chromedriver87.exe");
+        driver = new ChromeDriver();
+    }
+
+    @AfterMethod
+    public void signOffFromTrello() {
+//        System.setProperty("webdriver.chrome.driver", "C://Users//Office//Downloads//chromedriver_win32 (1)//chromedriver87.exe");
```

Comment:
If code is not needed - please remove it

Reply:
Removed